package.json

```diff
@@ -159,6 +159,7 @@
   },
   "scripts": {
     "bench": "node ./bin/run-bench.js",
+    "benchJson": "node ./bin/run-bench.js --json",
```

Comment:
im not sure i understand the need to have a diff script. can't the `bench` script just output json to a file? I find the previous default, although now enhanced, not very useful as the data just gets written to stdout.

Reply:
I could see wanting to debug by writing to console, but that's going to be the less common use case by far. I'll make output to file the default.